### PR TITLE
Allow to remove device information from junit report

### PR DESCRIPTION
**Pull request by acecilia**
Hi 👋 

This PR is an implementation of the discussions in https://github.com/XCTestHTMLReport/XCTestHTMLReport/pull/115

### CODE REVIEW

**Comment by acecilia** [comment]
@tylervick One detail: I noticed the version `2.2.2` was not updated in the source code during last release

File: Sources/XCTestHTMLReportCore/Classes/Models/JUnitReport.swift [hunk 1]
```diff
@@ -119,33 +119,38 @@ extension JUnitReport.TestResult: XMLRepresentable {
...
         case .systemErr:
-            return "        <system-err>\(title.stringByEscapingXMLChars)</system-err>"
+            return "    <system-err>\(title.stringByEscapingXMLChars)</system-err>"
         case .skipped:
-            return "        <skipped />"
+            return "    <skipped />"
         case .unknown:
             // falback to system-out. This is better than printing nothing
-            return "        <system-out>\(title.stringByEscapingXMLChars)</system-out>"
+            return "    <system-out>\(title.stringByEscapingXMLChars)</system-out>"
```
Comment (acecilia) [inline Sources/XCTestHTMLReportCore/Classes/Models/JUnitReport.swift:131]: Fixed indentation to what is used in the rest of the junit report: 2 spaces

**Comment by acecilia** [comment]
@tylervick 👋 🙂 Any chance of getting your opinion on this? 🙏

**Comment by nacho4d** [comment]
Great job!

**Comment by acecilia** [comment]
Adding it here for context: there was an alternative proposal of using `properties` tag to convey the platform information. I implemented it [here](https://github.com/XCTestHTMLReport/XCTestHTMLReport/pull/285), but in the end decided to discard it due to reasons explained [here](https://github.com/XCTestHTMLReport/XCTestHTMLReport/pull/285#issuecomment-1235647229)

**Review by tylervick — APPROVED** [review]
Awesome, thank you! Thank you for the detailed analysis w.r.t. junit modification.

**Comment by tylervick** [comment]
> @tylervick One detail: I noticed the version `2.2.2` was not updated in the source code during last release

🤦 we need a version bump job or something for this. It's unfortunate that SPM doesn't really have a good versioning paradigm..
I'll be sure to bump the version before releasing this :)

**Pull request merged**